pkg/config/templates/wicked-setup-bridge.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+ACTION=$1
+INTERFACE=$2
+
+case $ACTION in
+	pre-up)
+		# enable vlan-aware
+		ip link set $INTERFACE type bridge vlan_filtering 1
+		;;
+
+	post-up)
+		# inherit MAC address from {{ . }}
+		ip link set dev $INTERFACE address $(ip -json link show dev {{ . }} | jq -j '.[0]["address"]')
```

Comment:
`$(ip -json link show dev {{ . }} | jq -j '.[0]["address"]')`

please consider add check and log, to make sure the MAC is really got; and log essential info when failed

Reply:
move it to bond post up script